.github/workflows/rust.yml

```diff
@@ -1,18 +1,17 @@
 name: Rust
 
 on:
-  push:
-    branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
+
```

Comment:
Nit: Whitespace?

Reply:
resolved as not whitespaced fields, but values of a map.